app/src/main/java/com/courseproject/tindar/entities/Conversation.java

```diff
@@ -0,0 +1,56 @@
+package com.courseproject.tindar.entities;
+
+import java.util.ArrayList;
+
+public abstract class Conversation {
+    private String users1;
+    private String users2;
+    private ArrayList<Message> messages;
+    private String id;
+
+
+    //TODO complete abstract methods
+
+
+    public abstract void addMessage();
+    public abstract void deleteMessage();
+    public abstract void addReactionToMessage();
+    public abstract void flagMessage();
+
+    // Getter methods
+    public String getUsers1() {
+        return users1;
```

Comment:
for all get methods in this class can we use this.xxx to refer to the instance variable? e.g. return this.users1 instead of return users1. 

So the style would be consistent with the account class, and it would be more clear that we are referring to the instance variable.